server/routes/send.js

```diff
@@ -29,11 +29,13 @@ module.exports =
       asyncMiddleware(async (req, res) => {
         const { destination, bookingId } = req.params
         const { token, licence, prisoner } = res.locals
+        const { offenderNo } = prisoner
         const transition = transitionsForDestinations[destination]
 
         await notificationService.send({
           transition,
           bookingId,
+          offenderNo,
```

Comment:
is the offenderNo not already passed through as part of the prisoner below?  
Maybe you don't need to pass it separately?

Reply:
Yep - reverted this in latest commit